mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/OverviewFragment.java

```diff
@@ -130,22 +130,18 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         mSmartBlockingCount = (TextView) view.findViewById(R.id.cc_smart_blocking_count);
         mCliqzAttrackCount = (TextView) view.findViewById(R.id.cc_enhanced_tracking_count);
         mAdBlockCount = (TextView) view.findViewById(R.id.cc_enhanced_blocking_count);
-        mTrustSiteButton.setOnClickListener(this);
-        mRestrictSiteButton.setOnClickListener(this);
-        mPauseGhosteryButton.setOnClickListener(this);
-        mSmartBlockingSwitch.setOnCheckedChangeListener(this);
-        mAntiTrackingSwitch.setOnCheckedChangeListener(this);
-        mAdBlockingSwitch.setOnCheckedChangeListener(this);
```

Comment:
Are you sure about moving this around? It seems to me pointless if you already check the data availability in the listener.